cmake/std/atdm/sems-rhel7/environment.sh

```diff
@@ -46,14 +46,16 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL"* ]]; then
   if [[ "$ATDM_CONFIG_COMPILER" == "INTEL" ]] ; then
     export ATDM_CONFIG_COMPILER=INTEL-18.0.5
```

Comment:
Should the default intel version not be updated to intel-19.0.5 as well?  I think EMPIRE is the only real customer for this and I think only EMPIRE developers have been leaving the version number off of 'intel' in the build name when they set up a configuration.  I think other users tend to supply the exact Intel version.

Reply:
I'd be thrilled to make that switch, if that's what people want.

Reply:
```suggestion
    export ATDM_CONFIG_COMPILER=INTEL-19.0.5
```

Reply:
Maybe we should wait to make that change until after the build is available, and has been run through the more extensive set of tests that EMPIRE runs less often than on every change